app/controllers/admin/videos_controller.rb

```diff
@@ -0,0 +1,27 @@
+class Admin::VideosController < ApplicationController
+  before_action :require_admin
+
+  def new
+    @video = Video.new
+  end
+
+  def create
+    @video = Video.new(video_params)
+    if @video.valid?
+      @video.small_cover_url = @video.small_cover_upload.filename

```

Comment:
Why are you doing this? Your form should just submit the name your model expects and uses.